components/ToDoForm.js

```diff
@@ -0,0 +1,53 @@
+import styled from "styled-components";
+import Image from "next/image";
+import addButtonCircleActive from "../assets/addButtonCircleActive.svg";
+
+export default function ToDoForm({ onSubmitNewToDoItem, destinationId }) {
+  return (
+    <ToDoListForm
+      onSubmit={(event) => onSubmitNewToDoItem(event, destinationId)}
```

Comment:
It would be best if the onSubmitNewToDoItem function would only consume one parameter (todo). The destinationId would not have to be passed to this component and the form could handle the event logic (like preventdefault)